### ISSUE TITLE
Deserialization Error with the latest commit

### ISSUE THREAD

**Issue by Jasonthefirst** [matrixgpt/matrix-chatgpt-bot#245]
I updated the repository to the latest release 3.1.4 but now I all my matrix bots have the following error in the docker console: 
```
node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^
[Error: Failed to deserialize or serialize a JSON value missing field `version` at line 1 column 6903] {
  code: 'GenericFailure'
}
Node.js v20.9.0
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
I tried to downgrade but the error persists. Is there anything I can try? Do I need to reset my bots (deleting the storage)?

**Comment by bertybuttface** [comment]
Yup that is https://github.com/turt2live/matrix-bot-sdk/issues/321 you are running into.

Clearing the storage is the best way forward. I'm going to overhaul the storage to separate out the matrix stuff (keys etc) from the LLM history at some point. 

You should still be able to use a tagged release (3.1.4) you just can't use the latest development code without resetting.
